mesonwrap/wrapcreator.py

```diff
@@ -31,7 +31,7 @@
 
 # Replace this with proper parameterized callback if this need to be extended.
 _OUT_URL_BASE_DEFAULT = (
-    'https://wrapdb.mesonbuild.com/v1/projects/%s/%s/%d/get_zip')
```

Comment:
Please also change the call site on the line 87.

Reply:
Ok. Changed the call site on line 87. Any other issues?